code/modules/client/verbs/ooc.dm

```diff
@@ -80,6 +80,106 @@
 		GLOB.ooc_allowed = !GLOB.ooc_allowed
 	to_chat(world, "<B>The OOC channel has been globally [GLOB.ooc_allowed ? "enabled" : "disabled"].</B>")
 
+/client/verb/looc(msg as text)
+	set name = "LOOC"
+	set category = "OOC"
+
+	if(GLOB.say_disabled)	//This is here to try to identify lag problems
+		to_chat(usr, "<span class='danger'>Speech is currently admin-disabled.</span>")
+		return
+
+	if(!mob)
+		return
+
+	if(!holder)
+		if(!GLOB.looc_allowed)
+			to_chat(src, "<span class='danger'>LOOC is globally muted.</span>")
+			return
+		if(!GLOB.dlooc_allowed && (mob.stat == DEAD))
+			to_chat(usr, "<span class='danger'>LOOC for dead mobs has been turned off.</span>")
+			return
+		if(prefs.muted & MUTE_LOOC)
+			to_chat(src, "<span class='danger'>You cannot use LOOC (muted).</span>")
+			return
+		if(jobban_isbanned(src.mob, "OOC"))
+			to_chat(src, "<span class='danger'>You have been banned from LOOC.</span>")
+			return
+
+	msg = copytext(sanitize(msg), 1, MAX_MESSAGE_LEN)
+	var/raw_msg = msg
+
+	if(!msg)
+		return
+
+	msg = emoji_parse(msg)
+
+	if((copytext(msg, 1, 2) in list(".",";",":","#")) || (findtext(lowertext(copytext(msg, 1, 5)), "say")))
+		if(alert("Your message \"[raw_msg]\" looks like it was meant for in game communication, say it in LOOC?", "Meant for OOC?", "No", "Yes") != "Yes")
+			return
+
+	if(!holder)
+		if(handle_spam_prevention(msg, MUTE_LOOC))
+			return
+		if(findtext(msg, "byond://"))
+			to_chat(src, "<B>Advertising other servers is not allowed.</B>")
+			log_admin("[key_name(src)] has attempted to advertise in LOOC: [msg]")
+			message_admins("[key_name_admin(src)] has attempted to advertise in LOOC: [msg]")
+			return
+
+	if(!(prefs.chat_toggles & CHAT_LOOC))
+		to_chat(src, "<span class='danger'>You have LOOC muted.</span>")
+		return
+
+
+	log_talk(mob,"[key_name(src)] : [raw_msg]",LOGLOOC)
+	mob.log_message("[key]: [raw_msg]", INDIVIDUAL_LOOC_LOG)
+
+	var/list/clients_to_hear = list()
+
+	var/turf/looc_source = get_turf(mob.get_looc_source())
+	to_chat(world, "the looc source is called [looc_source.name], at [looc_source.x], [looc_source.y], [looc_source.z]")
+	var/list/stuff_that_hears = list()
+
+	for(var/mob/M in get_hear(7, looc_source))
+		stuff_that_hears += M
+
+	for(var/mob/M in stuff_that_hears)
+		to_chat(world, M.name)
+		if(!M.client && M.client_mobs_in_contents.len <= 0)
+			continue
+		if(M.client.prefs.chat_toggles & CHAT_LOOC)
+			clients_to_hear += M.client
+		for(var/mob/mob in M.client_mobs_in_contents)
+			if(mob.client.prefs.chat_toggles & CHAT_LOOC)
+				clients_to_hear += mob.client
+
+	for(var/client/C in GLOB.clients)
+		if(C in GLOB.admins)
+			if(C in clients_to_hear)
+				to_chat(C, "<span class='looc'><font color=LOOC_COLOR'>LOOC: [ADMIN_LOOKUPFLW(src)]: [msg]</font></span>")
```

Comment:
`span class='looc'` doesn't color the text, for some reason. Would love to help figuring out why.

Reply:
Tested and this is down to BYOND caching the .css files. Remove the `<font>` tags and clear your cache and you'll be 🆗